immuni_common/helpers/utils.py

```diff
@@ -52,3 +53,32 @@ def dense_dict(dictionary: Dict) -> Dict:
         for k, v in dictionary.items()
         if v is not None
     }
+
+
+# Note: This could be a Generic[T] with T as TypeVar. That would allow
+#  us to define a WeightedPair[str] and have the weighted_random
+#  return a valid typed str when used, but generic NamedTuples are not yet
+#  supported by mypy:
+#  https://github.com/python/mypy/issues/685
+class WeightedPair(NamedTuple):
+    """
+    Simple NamedTuple that allows us to describe
```

Comment:
_minor_: IMHO, the `Simple NamedTuple that allows us to ` is superfluous since (i) I'd usually avoid simple/complicated/and-similar since they may be subjective (not in this case, but I try to follow it if not strictly necessary), and (ii) it's clear we're talking about a NamedTuple since one line above we subclass it.